docs/content-guide/redirects.md

```diff
@@ -17,6 +17,29 @@ Use redirects for fun and profit!
 
 ---
 
+## Background
+
+If you move an existing article in the documentation, you **must** add redirects. Otherwise, links break between versions and users can't find the information they are looking for, even if it exists in the version they are viewing!
+
+If your product **is not** versioned (e.g., a cloud product), add **one** redirect:
+
+![Diagram showing a single path change and a single redirect](../images/single_redirect_diagram.png)
+
+This redirect ensures any requests to the old URL properly re-route to the new URL. To learn more, refer to [Example redirects](#example-redirects).
+
+If your product **is** versioned, add **three** redirects to reroute requests to the appropriate path in each version:
+
+![Diagram showing three redirects for a versioned product move](../images/three_redirects_per_versioned_move.png)
+
+The three redirects cover the following use cases:
+1. Redirects the latest version URL, which has no version path (i.e., `/product/<path>`).
+2. Redirects older versions to the old path (i.e., `/product/:version/<old_path>`).
```

Comment:
It's probably worth being explicit that your mapping the *new* path to the *old* path and noting that these redirects are needed explicitly for the version selector. Otherwise it kind of reads like you're creating redirects for old paths that haven't changed.